udata/forms/fields.py

```diff
@@ -297,7 +296,7 @@ def process_formdata(self, valuelist):
             self.data = valuelist
         elif valuelist:
             self.data = list(set([
-                x.strip().lower()
+                slugify(x, to_lower=True)
```

Comment:
I think we should factorize that in order to have the same tag rules everywhere
(ie. [here too](https://github.com/opendatateam/udata/blob/dev/udata/harvest/filters.py#L61-L62))